ui/src/app/pages/data/concept/concept-search.tsx

```diff
@@ -114,19 +114,17 @@ const ConceptSetMenu = ({
     closeOnClick
     content={
       <React.Fragment>
-        <TooltipTrigger>
```

Comment:
this wasn't valid.  was this a bug?

Reply:
looks like no.  I see that only the appropriate TooltipTriggers are rendered in both cases